src/main/kotlin/no/nav/syfo/api/endpoints/ArbeidsuforhetEndpoints.kt

```diff
@@ -30,6 +30,17 @@ fun Route.registerArbeidsuforhetEndpoints(
             this.veilederTilgangskontrollClient = veilederTilgangskontrollClient
         }
 
+        get(forhandsvarselPath) {
```

Comment:
Er det verdt å diskutere om vi skal ha et generelt GET-endepunkt for vurderinger i stedet? Da kan frontenden velge ut de som er av typen "FORHÅNDSVARSEL" for å vise riktig tilstand 🤔 
Bare ser for meg at vi etterhvert får et get-endepunkt som skal hente alle vurderinger som ender opp med å gjøre akkurat det samme (og de typene som ikke har noe varsel på seg vil bare ha `varsel = null`).

Reply:
OK, prøver meg på noen tilpasninger.